README.md

```diff
@@ -1,23 +1,33 @@
 # Assessing your React Learning
 
-- The purpose of this exercise is to get you used to being quizzed on _Interview Questions_ commonly asked about ReactJS.
-- Answers to your written questions will be recorded in _Answers.md_
-- This is to be worked on alone but you can use outside resources. You can _reference_ any old code you may have, and the React Documentation, however, please refrain from copying and pasting any of your answers. Try and understand the question and put your responses in your own words. Be as thorough as possible when explaining something.
+X The purpose of this exercise is to get you used to being quizzed on _Interview Questions_ commonly asked about ReactJS.
+X Answers to your written questions will be recorded in _Answers.md_
+X This is to be worked on alone but you can use outside resources. You can _reference_ any old code you may have, and the React Documentation, however, please refrain from copying and pasting any of your answers. Try and understand the question and put your responses in your own words. Be as thorough as possible when explaining something.
 
 ## Start by forking and cloning this repository.
 
 ## Questions
 
 1.  What is React JS and what problems does it try and solve?
 
+React JS is a Javascript library a developer created that allows us to build out websites with good UI more quickly.
+
 1.  What does it mean to _think_ in react?
 
+To consider projects, as you start them, in components. Separating each section of a page down into boxes of components, organized by like functions and design.
```

Comment:
Exactly! 👍 

I noticed you put your answers in the README. Next time try putting them in the Answers.md file.